src/lib/src/datepicker/datepicker.component.spec.ts

```diff
@@ -92,4 +92,11 @@ describe('Component: DatepickerComponent', () => {
     fix.detectChanges();
     expect(comp.picker.getDate().getDate()).toEqual(1);
   });
+
+  it('should support formatting the date', () => {
```

Comment:
We can fix datepicker formatting in Publish now too: https://github.com/PRX/publish.prx.org/issues/458

Reply:
:+1: